drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/test/java/org/drools/workbench/services/verifier/plugin/client/fromfile/DatesFromFileTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.services.verifier.plugin.client.fromfile;
+
+import org.drools.verifier.api.reporting.CheckType;
+import org.drools.verifier.api.reporting.Severity;
+import org.drools.verifier.core.main.Analyzer;
+import org.drools.workbench.models.guided.dtable.backend.GuidedDTXMLPersistence;
+import org.drools.workbench.models.guided.dtable.shared.model.GuidedDecisionTable52;
+import org.drools.workbench.services.verifier.plugin.client.AnalyzerUpdateTestBase;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.drools.workbench.services.verifier.plugin.client.testutil.TestUtil.assertContains;
+import static org.drools.workbench.services.verifier.plugin.client.testutil.TestUtil.assertDoesNotContain;
+import static org.drools.workbench.services.verifier.plugin.client.testutil.TestUtil.loadResource;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DatesFromFileTest extends AnalyzerUpdateTestBase {
```

Comment:
Thank you for this test @Rikkola , I tried to open GDST in decision central, I can confirm validation works but reports missing range in inappropriate order.

See this PR https://github.com/kiegroup/kie-wb-common/pull/3569

Reply:
I need to cherry-pick this for the 7.48.x bundle